code/edge/src/main/java/com/adobe/marketing/mobile/EdgeHitProcessor.java

```diff
@@ -151,6 +152,18 @@ public void onComplete() {
 			edgeHit.getRequestId()
 		);
 
+		if (!isValidUrl(url)) {
+			Log.warning(
```

Comment:
you can skip this log based on the other logs inside isValidUrl

Reply:
This is a `warning` stating the event is being dropped. The other two are `debug` logs stating why the URL is invalid. I did it this way because of context and responsibility. The `isValidUrl` is just validating the URL while `sendNetworkRequest` is responsible for dropping the request if the url is invalid.